app/models/post.rb

```diff
@@ -2,4 +2,12 @@ class Post < ApplicationRecord
   belongs_to :author, class_name: 'User'
   has_many :likes
   has_many :comments
+
+  def update_posts_counter(_count)
+    author.increment!(:post_counter)
+  end
+
+  def recent_comments(limit = 5)
+    comments.last(limit)
+  end
```

Comment:
- [ ] Kindly consider making use of one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_posts_counter` method every time a new post has been saved to the database, then make the `update_post_counter` method private

e.g

```ruby
 after_save :update_post_counter

  private

  def update_posts_counter
    author.increment!(:posts_counter)
  end
```